docassemble/RFApackage/data/questions/RFApackage.yml

```diff
@@ -1101,16 +1181,53 @@ fields:
     maxlength: 34
     required: False
+    js show if: |
+      val("existing_order['existing_order_divorce']") || val("existing_order['dissolution']") || val("existing_order['existing_order_RFA']") || val("existing_order['criminal_proceeding']") || val("existing_order['parentage_order']") || val("existing_order['guardianship_order']") || val("existing_order['juvenile_order']") || val("existing_order['child_support_order']") 
+      
   - "In what county/counties is the existing order or a pending court proceeding?": existing_order_county
     maxlength: 36
     required: False
+    js show if: |
+      val("existing_order['existing_order_divorce']") || val("existing_order['dissolution']") || val("existing_order['existing_order_RFA']") || val("existing_order['criminal_proceeding']") || val("existing_order['parentage_order']") || val("existing_order['guardianship_order']") || val("existing_order['juvenile_order']") || val("existing_order['child_support_order']") 
+      
   - "What is the name of your attorney for the existing order or a pending court proceeding -- if you have one?": existing_order_users_attorney
     maxlength: 25
     required: False
+    js show if: |
+      val("existing_order['existing_order_divorce']") || val("existing_order['dissolution']") || val("existing_order['existing_order_RFA']") || val("existing_order['criminal_proceeding']") || val("existing_order['parentage_order']") || val("existing_order['guardianship_order']") || val("existing_order['juvenile_order']") || val("existing_order['child_support_order']")
+      
   - "What is the name of the other attorney for the existing order or a pending court proceeding -- if the defendant has one?": existing_order_OP_attorney
     maxlength: 25
     required: False
-
+    js show if: |
+      val("existing_order['existing_order_divorce']") || val("existing_order['dissolution']") || val("existing_order['existing_order_RFA']") || val("existing_order['criminal_proceeding']") || val("existing_order['parentage_order']") || val("existing_order['guardianship_order']") || val("existing_order['juvenile_order']") || val("existing_order['child_support_order']") 
+---
+needs:
+  - existing_order_text
+variable name: existing_order_text
+use objects: True
+data:
+  existing_order_divorce:
+    question: For divorce or separation?
+    document: For divorce or separation?
+  existing_order_RFA:
+    question: For a civil union dissolution?
+    document: For a civil union dissolution?       
+  criminal_proceeding:
+    question: For a criminal matter?  
+    document: For a criminal matter?
+  parentage_order:
+    question: For parentage?  
+    document: For parentage?
+  guardianship_order:
+    question: For guardianship?  
+    document: For guardianship?
+  juvenile_order:
+    question: For a juvenile (DCF) matter?
+    document: For a juvenile (DCF) matter?
+  child_support_order: 
+    question: For child support?
+    document: For child support?
 ---
 id: About your home
```

Comment:
Expected appearance should be a choices list of:

Rent or own?

-the home is rented
-the home is owned
-I don't know

In whose name?

-rented or owned by ${ plaintiff } and ${ defendant }
-rented or owned by ${ plaintiff }
-rented or owned by ${ defendant }
-rented or owned by someone else
-I don't know